deque.js

```diff
@@ -346,8 +346,17 @@ Deque.prototype.lastIndexOf = function (value, index) {
     return -1;
 }
 
-// TODO rename findValue
-Deque.prototype.find = function (value, equals, index) {
+Deque.prototype.find = function () {
+    if (
+        typeof console !== 'undefined' &&
+            typeof console.warn === 'function'
+    ) {
+        console.warn('.find function is deprecated please use findValue instead.');
```

Comment:
Should only leave this note once for the module’s lifetime.

Reply:
added deprecatedWarn in each file for now, with TODO remove in v6 notes